src/System.Security.Cryptography.Algorithms/tests/System.Security.Cryptography.Algorithms.Tests.csproj

```diff
@@ -142,7 +144,7 @@
     <Compile Include="TripleDesTests.cs" />
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Common\Interop\Unix\Interop.Libraries.cs</Link>
-    </Compile>
+    </Compile> 
```

Comment:
This is due to the merge with master. There was a build break I had to fix locally (adding the package reference) but it was also fixed in master and I missed this when I merged.

It makes sense to undo the changes to this file.